services/algolia-sync/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@base-cms/algolia-sync",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "author": "Jeremy Laird <jlaird@endeavorb2b.com>",
+  "repository": "https://github.com/base-cms/base-cms/tree/master/services/algolia-sync",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "node src/index.js",
+    "lint": "eslint --ext .js --max-warnings 5 ./",
+    "test": "yarn lint"
+  },
+  "dependencies": {
+    "@base-cms/graphql-fragment-types": "^1.9.0",
+    "algoliasearch": "^4.0.0",
+    "apollo-cache-inmemory": "^1.6.5",
+    "apollo-client": "^2.6.8",
+    "apollo-link-http": "^1.5.16",
+    "aws-sdk": "^2.610.0",
+    "cli-progress": "^3.6.0",
+    "dotenv": "^8.2.0",
+    "graphql": "^14.6.0",
+    "graphql-tag": "^2.10.1",
+    "minimist": "^1.2.0",
```

Comment:
Other services in the monorepo use `yargs` -- prefer if the same packages are used across the entire repo, unless there are specific featuresets that don't translate between them.

Reply:
I switch it to yargs like you asked